sample_configs/bootstrap.sh

```diff
@@ -66,3 +67,7 @@ su - postgres -c "cat /vagrant/sample_configs/postgres.sql | psql"
 
 # Start httpd back up
 service apache2 start
+
+# Somewhere to store audio uploads
+mkdir -p /music/records
```

Comment:
Right now, that change just makes things work as they are. Editing the default should be its own issue/change.